### PR TITLE
Add moments page for Facebook container

### PR DESCRIPTION
**Pull request by ncloudioj**
This adds a new Moments Page for Facebook Container and retires the WNP_MOMENTS_8 as it shares the profile age targeting with the new one.

The detailed targeting is documented [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1656958#c0).

### CODE REVIEW

File: moments.yaml [hunk 1]
```diff
@@ -47,3 +28,23 @@
...
+  groups: [moments-pages]
+  content:
+    action:
+      data:
+        expireDelta: 604800000
+        url: https://www.mozilla.org/firefox/welcome/7/
+      id: moments-wnp
+    bucket_id: WNP_MOMENTS_7
+  # Fx 70+, profile age > 30 days all en-* and selected locales
+  targeting: firefoxVersion >= 69 &&
```
Comment (piatra) [inline moments.yaml:41]: nit: comment says 70+

File: moments.yaml [hunk 2]
```diff
@@ -47,3 +28,23 @@
...
+    (
+      localeLanguageCode == "en" ||
+      locale in ["be", "cy", "da", "de", "dsb", "es-AR", "es-CL", "es-ES", "es-MX", "fr", "fy-NL", "gn", "hr", "hsb", "hu", "hy-AM", "ia", "id", "it", "ka", "kab", "nl", "nn-NO", "pl", "pt-BR", "pt-PT", "ro", "ru", "sk", "sl", "sq", "sr", "sv-SE", "tr", "uk", "vi", "zh-TW"]
+    ) && ((currentDate|date - profileAgeCreated) / 86400000) > 30
+    &&  xpinstallEnabled &&
+    (['@contain-facebook','{bb1b80be-e6b3-40a1-9b6e-9d4073343f0b}', '{a50d61ca-d27b-437a-8b52-5fd801a0a88b}'] intersect addonsInfo.addons|keys)|length == 0
```
Comment (piatra) [inline moments.yaml:47]: I'm not sure what those other 2 addons are, I could only confirm `Facebook Container` has the `@contain-facebook` id.

Comment (piatra) [reply moments.yaml:47]: Nvm: we should keep that list, it looks good. I found the trick `addons.mozilla.org/firefox/addon/<guid>`

Comment (ncloudioj) [reply moments.yaml:47]: Yeah, I wasn't sure either at first. But looks like all the Facebook Container CFRs have those two included.

**Pull request merged**